README.md

```diff
@@ -375,13 +375,47 @@ Edit this line and replace `127.0.1.1` with your public IP.
 
 ## Example Playbook
 
-This is an example, of how to use this role. Warning: the value of the Variables should be changed!
+This is an example of how to use this role. *Warning:* the values of the variables should be changed!
 
-```yaml
-    - hosts: servers
-      roles:
-         - { role: ebbba.bigbluebutton, bbb_coturn_secret: ee8d093109a9b273, bbb_greenlight_secret: 107308d54ff4a5f, bbb_greenlight_db_password: 2585c27c785e8895ec, bbb_letsencrypt_email: mail@example.com }
+Assuming the following directory structure:
+```
+├── ansible
+    ├── roles
+    │   └── ansible-role-bigbluebutton
```

Comment:
I would prefer this to follow the ansible-fqdn spec, e.g.: `ebbba.bigbluebutton`

Reply:
```suggestion
    │   └── ebbba.bigbluebutton
```

Reply:
I added an extra step to clone the repository in the right place and changed the directory structure as per your suggestion.